### PR TITLE
README.md: Typo fixes

### PR DESCRIPTION
**Pull request by yennster**
Fixed spelling typos.

### CODE REVIEW

**Comment by screamerbg** [comment]
@yennster This one as well. Sorry :(

**Comment by screamerbg** [comment]
@yennster Bump

**Comment by yennster** [comment]
@screamerbg Done!

File: README.md [hunk 1]
```diff
@@ -285,8 +285,13 @@ Methods for configuring toolchains that appear earlier in the above list overrid
...
 * To use the [IAR EWARM toolhain](https://www.iar.com/iar-embedded-workbench/#!?architecture=ARM), set `IAR_PATH` to the *base* directory of your IAR installation. Use versions 7.80 of IAR EWARM; prior versions might be incompatible with the tools.
+=======
+* To use the [GCC ARM Embedded toolchain](https://launchpad.net/gcc-arm-embedded), set `GCC_ARM_PATH` to the *binary* directory of your GCC ARM installation (example: C:\Program Files\GNU Tools ARM Embedded\6 2017q2\bin). Use version 6 of GCC ARM Embedded; version 5.0 or any older version might be incompatible with the tools.
+* To use the [IAR EWARM toolchain](https://www.iar.com/iar-embedded-workbench/#!?architecture=ARM), set `IAR_PATH` to the *base* directory of your IAR installation. Use versions 7.80 of IAR EWARM; prior versions might be incompatible with the tools.
+>>>>>>> Fixed spelling typo
```
Comment (bridadan) [inline README.md]: Looks like a conflict made it in, can you fix this?

Comment (yennster) [inline README.md]: @bridadan Oops, yep!

**Comment by yennster** [comment]
@bridadan Should be good now

**Comment by screamerbg** [comment]
@yennster This one has merge errors. Could you please rebase?

**Comment by screamerbg** [comment]
Fixed it myself. Hope you don't mind :)

**Comment by yennster** [comment]
@screamerbg What errors were still present? Sorry, I am new to pull requests 😋

**Pull request merged**